move/Move.toml

```diff
@@ -4,7 +4,10 @@ version = '1.0.0'
 upgrade_policy = "compatible"
 
 [addresses]
-aptogotchi = '573503bbc3bc51ca5ff71042c548babfe604239c08c6fcd91e1a71ef44716c92'
+aptogotchi = '_'
+
+[dev-addresses]
+aptogotchi = "0x100"
```

Comment:
learned this trick from max that we can have dev address for testing so we don't need to pass it in `move test ...`

Reply:
can you add a comment somewhere in the code to explain it? it'll be useful for others to know

Reply:
done